### ISSUE TITLE
numpy.randint fails when asked to generate int64 [py3.5, np1.18.4]

### ISSUE THREAD

**Issue by drewrisinger** [pyGSTio/pyGSTi#138]
**Describe the bug**
I get the following error message while running the following code:
#### Error message (truncated)
```python
- Sampling 4 circuits at DRB length 0 (1 of 5 depths)
  - Number of circuits sampled =

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
 in 
     25     pspec,
     26     depths,
---> 27     4,
     28 )

c:\...\github\pygsti\pygsti\protocols\rb.py in __init__(self, pspec, depths, circuits_per_depth, qubit_labels, sampler, samplerargs, addlocal, lsargs, randomizeout, cliffordtwirl, conditionaltwirl, citerations, compilerargs, partitioned, descriptor, add_default_protocol, seed, verbosity)
    368                     cliffordtwirl=cliffordtwirl, conditionaltwirl=conditionaltwirl,
    369                     citerations=citerations, compilerargs=compilerargs,
--> 370                     partitioned=partitioned)
    371                 circuits_at_depth.append(c)
    372                 idealouts_at_depth.append((''.join(map(str, iout)),))

c:\...\github\pygsti\pygsti\algorithms\randomcircuit.py in direct_rb_circuit(pspec, length, qubit_labels, sampler, samplerargs, addlocal, lsargs, randomizeout, cliffordtwirl, conditionaltwirl, citerations, compilerargs, partitioned)
   1329     if cliffordtwirl:
   1330         # Sample a uniformly random Clifford.
-> 1331         s_initial, p_initial = _symp.random_clifford(n)
   1332         # Find the composite action of this uniformly random clifford and the random circuit.
   1333         s_composite, p_composite = _symp.compose_cliffords(s_initial, p_initial, s_rc, p_rc)

c:\...\github\pygsti\pygsti\tools\symplectic.py in random_clifford(n)
   1477 
   1478     """
-> 1479     s = random_symplectic_matrix(n, convention='standard')
   1480     p = random_phase_vector(s, n)
   1481 

c:\...\github\pygsti\pygsti\tools\symplectic.py in random_symplectic_matrix(n, convention)
   1446 
   1447     """
-> 1448     index = random_symplectic_index(n)
   1449     s = get_symplectic_matrix(index, n)
   1450 

c:\...\github\pygsti\pygsti\tools\symplectic.py in random_symplectic_index(n)
   1949             for i in range(0, n):
   1950                 add = zeros_string(m)
-> 1951                 sample = _np.random.randint(10**digits2)
   1952                 for j in range(0, i):
   1953                     add += zeros_string(digits2)

mtrand.pyx in numpy.random.mtrand.RandomState.randint()

_bounded_integers.pyx in numpy.random._bounded_integers._rand_int32()

ValueError: high is out of bounds for int32
```
#### Sample Code
```python
import pygsti
gates = ["Gi", "Gxpi2", "Gxpi", "Gxmpi2", "Gypi2", "Gypi", "Gympi2", "Gzpi2", "Gzpi", "Gzmpi2"]
pspec = pygsti.obj.ProcessorSpec(
    10,
    gates,
    availability={},
    verbosity=10,
    construct_clifford_compilations={"absolute": ("paulis", "1Qcliffords")},
    construct_models=("clifford",),
)
depths = [0, 2, 4, 8, 16]
edesign = pygsti.protocols.DirectRBDesign(pspec, depths, 4)
```

**To Reproduce**
Run sample code above against commit 6d0722d65f1ca4a1352498365a8089b5822691cb

**Expected behavior**
No errors

**Code Changes that resolve**:
Traced to ``numpy.random.randint()`` calls in: https://github.com/pyGSTio/pyGSTi/blob/dd4ad669931c7f75e026456470cf33ac5b682d0d/pygsti/tools/symplectic.py#L1951-L1959

## Resolution:
It runs by changing these to:
```python
numpy.random.randint(..., dtype=np.int64)
```

**Environment (please complete the following information):**
 - pyGSTi version: commit 6d0722d65f1ca4a1352498365a8089b5822691cb
 - python version: 3.5
 - OS: Windows 10

Conda environment (stripped for brevity):
```
# packages in environment at C:\Users\drewr\miniconda3\envs\py35-artiq4:
#
# Name                    Version                   Build  Channel
cython                    0.28.5           py35h6538335_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
defusedxml                0.6.0                      py_0    conda-forge
entrypoints               0.2.3                    py35_2    conda-forge
h5py                      2.8.0            py35hb24199d_2    conda-forge
hdf5                      1.10.2               he51fdeb_2    conda-forge
httplib2                  0.17.3                   pypi_0    pypi
jupyter                   1.0.0                      py_2    conda-forge
matplotlib                2.2.5                    pypi_0    pypi
mkl                       2018.0.3                      1
mkl_fft                   1.0.9                    py35_0    conda-forge
mkl_random                1.0.1                    py35_0    conda-forge
more-itertools            8.2.0                    pypi_0    pypi
mpmath                    1.1.0                    pypi_0    pypi
multidict                 4.4.2            py35hfa6e2cd_0    conda-forge
networkx                  2.4                      pypi_0    pypi
numpy                     1.18.4                   pypi_0    pypi
pandas                    0.23.4           py35h830ac7b_0    conda-forge
pandoc                    2.9.2.1                       0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
pint                      0.9                        py_0    conda-forge
pip                       20.1.1             pyh9f0ad1d_0    conda-forge
plotly                    4.6.0                    pypi_0    pypi
ply                       3.11                     pypi_0    pypi
prettytable               0.7.2                      py_3    conda-forge
prometheus_client         0.8.0              pyh9f0ad1d_0    conda-forge
prompt_toolkit            2.0.10                     py_0    conda-forge
pygsti                    0.9.9.2.post156+g2f3cbc2b           dev_0    <develop>
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.9.2            py35h6538335_2
pyqtgraph                 0.10.0                     py_7    conda-forge
pytables                  3.4.4            py35h7e76658_1    conda-forge
python                    3.5.5                he025d50_2    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
pytz                      2020.1             pyh9f0ad1d_0    conda-forge
pywin32                   223             py35hfa6e2cd_1000    conda-forge
pywinpty                  0.5.4                    py35_2    conda-forge
regex                     2018.08.29       py35hfa6e2cd_0    conda-forge
scipy                     1.4.1                    pypi_0    pypi
simplegeneric             0.8.1                      py_1    conda-forge
sip                       4.19.8          py35h6538335_1000    conda-forge
sortedcontainers          2.1.0                    pypi_0    pypi
statsmodels               0.9.0                    py35_0    conda-forge
sympy                     1.5.1                    pypi_0    pypi
uncertainties             3.1.2                    pypi_0    pypi
```

**Comment by enielse** [comment]
Fixed by 86ad0f595c6a72c774eb966bceb60b616e97a30d.  Let's leave this issue open until this commit makes it into a release.

**Comment by enielse** [comment]
Version 0.9.10 fixes this issue.
